### DISCUSSION TITLE
Put some custom content in right bar?

### DISCUSSION THREAD

**Discussion by alisman** [retypeapp/retype#317] [Q&A]
Is this possible to achieve?  I want to put some custom content in the right bar of the page so it appears above or below the list of internal links that appears (when there are such)

![image](https://user-images.githubusercontent.com/186521/162327226-e05a2dbe-118a-4e09-bc86-cc17b007247b.png)

**Comment by geoffreymcgill** [comment]
It's not possible right now, but that is a good idea. 

Would you want custom content on each page? or the same content on all pages? Likely both I guess.

I'll have to think how and where this content would be configured. I can't think of an obvious solution at the moment, so I'll chew on it to try and come up with some option(s). Any suggestions you have would be appreciated.

**Reply by alisman** [reply]
HI Geoff, I think both page specific and global would probably be nice.    What about a markdown block similar to container which will just be dumped in the right bar?   So it would just sit in the page files.

**Reply by geoffreymcgill** [reply]
We are discussing the functionality of this feature. To help with the discussion, can you provide an example or two of content that you would like to render in the right sidebar? We have some ideas of what might typically be rendered in that column, but just would be nice to hear from your perspective.

**Comment by alisman** [comment]
This kind of thing.  But I'm sure people, including myself, would come up
with all manner of things to put there, so I would advocate for full
flexibility.
[image: image.png]
